mrp_bom_widget_section_and_note_one2many/views/view_mrp_bom.xml

```diff
@@ -0,0 +1,131 @@
+                        <field name="parent_product_tmpl_id" invisible="1" />
+                        <field
+                                name="product_uom_id"
+                                options="{'no_open':True,'no_create':True}"
+                                groups="uom.group_uom"
+                                attrs="{
+                                   'required': [('display_type', '=', False)],
+                                   'invisible': [('display_type', '=', True)],
+                                }"
+                            />
+                        <field
+                                name="possible_bom_product_template_attribute_value_ids"
+                                invisible="1"
+                            />
+                        <field
+                                name="bom_product_template_attribute_value_ids"
+                                optional="hide"
+                                widget="many2many_tags"
+                                options="{'no_create': True}"
+                                attrs="{
+                                'column_invisible': [('parent.product_id', '!=', False)],
+                                'invisible': [('display_type', '=', True)]
+                                }"
+                                groups="product.group_product_variant"
+                            />
+                        <field name="allowed_operation_ids" invisible="1" />
+                        <field
+                                name="operation_id"
+                                groups="mrp.group_mrp_routings"
+                                optional="hidden"
+                                attrs="{
+                                  'column_invisible': [('parent.type','not in', ('normal', 'phantom'))],
+                                  'invisible': [('display_type', '=', True)]
+                                  }"
+                                options="{'no_quick_create':True,'no_create_edit':True}"
+                            />
+                        <field name="manual_consumption_readonly" invisible="1" />
+                        <field
+                                name="manual_consumption"
+                                optional="hide"
+                                width="1.0"
+                                attrs="{
+                                  'readonly': [('manual_consumption_readonly', '=', True)],
+                                  'invisible': [('display_type', '=', True)],
+                                  }"
+                                force_save="1"
+                            />
+                    </tree>
+                </field>
+            </page>
+          </xpath>
+          <!-- Hide the original page -->
+          <xpath expr="//page[2]" position="attributes">
+              <attribute name="groups">base.group_erp_manager</attribute>
+          </xpath>
```

Comment:
Hum. This one is OK but it is making test failing. see : https://github.com/OCA/manufacture/actions/runs/9745619815/job/26894022246?pr=1197#step:8:558

maybe try : 
``              <attribute name="invisible">1</attribute>``

Not sure.

Reply:
>  After all, maybe it's better not hiding this and letting every one choosing what to do in custum module ?

Do you think it can generate errors if the user manipulate the original field ? if yes, we can consider to set the field readonly. don't you think ?

Reply:
So let's make the CI green again !

Reply:
💚 🟢 🟩 🥬 🥑 🌳 🥦 ♻️ 🐸 ✅ 💚

Reply:
i love a good green ~~salad~~ CI !